pkg/sql/catalog/descpb/privilege_test.go

```diff
@@ -157,7 +158,7 @@ func TestPrivilege(t *testing.T) {
 				tcNum, descriptor, show, tc.show)
 		}
 		for i := 0; i < len(show); i++ {
-			if show[i].User != tc.show[i].User || show[i].PrivilegeString() != tc.show[i].PrivilegeString() {
+			if !reflect.DeepEqual(show[i], tc.show[i]) {
```

Comment:
`PrivilegeString()` was only being used by this test.  

Instead of coming up with a string representation for the struct, I changed the test to use `DeepEqual`.